### PR TITLE
really really ready to publish now

### PR DESCRIPTION
**Pull request by nicklan**
Needed to add descriptions.

Also removed authors, it's not needed, and projects with many authors like this tend to leave it out.

### CODE REVIEW

File: ffi/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "delta_kernel_ffi"
```
Comment (rtyler) [inline ffi/Cargo.toml:2]:
```suggestion
name = "delta-kernel-ffi"
```

File: kernel/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "delta_kernel"
```
Comment (rtyler) [inline kernel/Cargo.toml:2]:
```suggestion
name = "delta-kernel"
```

**Review by rtyler — APPROVED** [review]
@nicklan said in slack that he and @zachschuermann love this yucky underscores in their crate names. I guess since my name is no longer directly affiliated with the crate once it's published, I can approve :smiling_imp:

**Review by tdas — APPROVED** [review]

**Comment by tdas** [comment]
YAYYYYYYYYYYYYYYYYYYY 🚀 🥳

**Comment by nicklan** [comment]
> @nicklan said in slack that he and @zachschuermann love this yucky underscores in their crate names. I guess since my name is no longer directly affiliated with the crate once it's published, I can approve 😈

I apologize for my practical decision that the `use delta_kernel::...` code should match the actual crate name ;)

**Pull request merged**